config/karma.conf.ts

```diff
@@ -1,117 +1,116 @@
+// eslint-disable-next-line rulesdir/disallow-fec-relative-imports
```

Comment:
@Hyperkid123 FYI I had to ignore this suggested change (at least until we upgrade the version of `frontend-components-config` in package.json) as the rule was trying to suggest `@redhat-cloud-services/frontend-components-config/config` however that does not exist (in this version the config function is exported at the root).